pallets/elections/src/tests.rs

```diff
@@ -0,0 +1,20 @@
+#![cfg(test)]
+
+use frame_election_provider_support::{ElectionProvider, Support};
+
+use crate::mock::*;
+
+#[test]
+fn test_elect() {
+    new_test_ext(vec![1, 2]).execute_with(|| {
+        let elected = <Elections as ElectionProvider<AccountId, u64>>::elect();
+        assert!(elected.is_ok());
+
+        let supp = Support {
+            total: 0,
+            voters: Vec::new(),
```

Comment:
Could you add a test that outputs something nontrivial?

Reply:
I would expect so, but if it would be lots of work then feel free to resolve this instead.